src/emulator/functionsEmulator.ts

```diff
@@ -767,27 +779,31 @@ export interface InvokeRuntimeOpts {
   env?: { [key: string]: string };
   ignore_warnings?: boolean;
 }
+
 export function InvokeRuntime(
```

Comment:
why is this function `InvokeRuntime` rather than `invokeRuntime`? the former is indicative of a constructor, which would require `new`.

Reply:
I was trying to be consistent with the style Abe had been using in this file but I agree, should be consistent with the CLI at large.